sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2023-2024.bpmn

```diff
@@ -595,6 +565,48 @@
     </bpmn:intermediateThrowEvent>
     <bpmn:sequenceFlow id="Flow_15xhu93" sourceRef="Event_0rpww4j" targetRef="Event_1mkiqso" />
     <bpmn:sequenceFlow id="Flow_1idr6ms" sourceRef="Event_1mkiqso" targetRef="Event_0hxxmu9" />
+    <bpmn:businessRuleTask id="Activity_134pojh" name="dmnPartTimeAwardFamilySizeVariables">
+      <bpmn:extensionElements>
+        <zeebe:calledDecision decisionId="dmnPartTimeAwardFamilySizeVariables" resultVariable="dmnPartTimeAwardFamilySizeVariables" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_14w7mr0</bpmn:incoming>
+      <bpmn:outgoing>Flow_16zqfik</bpmn:outgoing>
+    </bpmn:businessRuleTask>
+    <bpmn:businessRuleTask id="Activity_0qzkp53" name="dmnPartTimeAwardAllowableLimits">
+      <bpmn:extensionElements>
+        <zeebe:calledDecision decisionId="dmnPartTimeAwardAllowableLimits" resultVariable="dmnPartTimeAwardAllowableLimits" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0jntyac</bpmn:incoming>
+      <bpmn:outgoing>Flow_14w7mr0</bpmn:outgoing>
+    </bpmn:businessRuleTask>
+    <bpmn:intermediateThrowEvent id="Event_0re3zsg" name="Calculate calculatedDataChildCareCost">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if &#10;  dmnPartTimeProgramYearMaximums.limitWeeklyChildCare * offeringWeeks &#60; studentDataDaycareCosts11YearsOrUnder + studentDataDaycareCosts12YearsOrOver&#10;then&#10;  dmnPartTimeProgramYearMaximums.limitWeeklyChildCare * offeringWeeks&#10;else&#10;  studentDataDaycareCosts11YearsOrUnder + studentDataDaycareCosts12YearsOrOver" target="calculatedDataChildCareCost" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0lp9mvw</bpmn:incoming>
+      <bpmn:outgoing>Flow_0temot0</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_1j6ep9h" name="calculatedDataTotalAssessedNeed">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=sum([&#10;  calculatedDataActualTuitionCosts,&#10;  calculatedDataMandatoryFees,&#10;  calculatedDataProgramRelatedCosts,&#10;  calculatedDataTransportationAllownace,&#10;  calculatedDataMiscellaneousAllowance,&#10;  calculatedDataTotalChildCareCost&#10;][item != null])" target="calculatedDataTotalAssessedNeed" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1mrmz3w</bpmn:incoming>
+      <bpmn:outgoing>Flow_0jntyac</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_1mrmz3w" sourceRef="Event_1pr211h" targetRef="Event_1j6ep9h" />
+    <bpmn:intermediateThrowEvent id="Event_1pr211h" name="Calculate Total Childcare cost">
```

Comment:
Correct me if I'm wrong. I thought we needed to update the "Calculate calculatedDataChildCareCost" event to the name "Calculate Child Care Costs" and include `calculatedDataTotalChildCareCost` in the output for the expected calculation. Image taken from the story:
![image](https://github.com/bcgov/SIMS/assets/148148914/dfb123c1-ad89-4f69-a060-a293808d1be4)

I understand the solutions in the story are there for guidance. I'm good with this approach if we all agree to use a new Intermediate Throw Event named "Calculate Child Care Costs" to do the calculation instead of the screenshot above. Give a 👍 if you agree: @andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993

Reply:
@lewischen-aot my understanding is that the goal of this first PR is to have the variable created and the changes needed in the `assessment-gateway` implemented to allow other tickets in the same sprint to do also possible changes in `assessment-gateway`.
@dheepak-aot correct me if I am wrong.
I intend to take a closer look at the `part-time-assessment-*.bpmn` once the implementation is completed.

Reply:
Thanks @andrewsignori-aot. I'm good here with creating a new event to do the calculation for `calculatedDataTotalChildCareCost` later on.

Reply:
You are right @andrewsignori-aot . @lewischen-aot this is not the final state for part-time-assessment*.bpmn. we added a throw event to include a variable to return assessment gateway.

Reply:
I understand it now, thanks all for the explanation. I was in a similar thought as @lewischen-aot